app/models/inventory.rb

```diff
@@ -0,0 +1,10 @@
+class Inventory < ApplicationRecord
+  belongs_to :phone
```

Comment:
if we delete a phone, should we also delete inventory?

Reply:
Added dependent destroy on `phone` and `model` model